components/feature/findinpage/src/main/java/mozilla/components/feature/findinpage/FindInPageView.kt

```diff
@@ -39,10 +38,10 @@ class FindInPageView @JvmOverloads constructor(
     var onCloseButtonPressListener: (() -> Unit)? = null
 
     /**
-     * Gives access to the underneath engine, to start the search and to control the navigation between
+     * Gives access to the underneath sessionEngine to start the search and to control the navigation between
```

Comment:
nit: Comment still references `sessionEngine`. So, should be "..access to the engine session to start...."